app/_data/docs_nav_konnect.yml

```diff
@@ -183,8 +183,10 @@
           url: /dev-portal/access-and-approval/add-teams/
         - text: Manage Application Registrations
           url: /dev-portal/access-and-approval/manage-app-connections/
-        - text: Configure SSO for Dev Portal
+        - text: Configure generic SSO for Dev Portal
           url: /dev-portal/access-and-approval/sso/
+        - text: Configure Okta SSO 
```

Comment:
Should Dev Portal be mentioned in this title too?

Reply:
Good catch! I'll add that

Reply:
```suggestion
        - text: Configure Okta SSO for Dev Portal
```